### PR TITLE
sysvinit service module

### PR DESCRIPTION
**Pull request by maxamillion**
Signed-off-by: Adam Miller <admiller@redhat.com>

##### SUMMARY

New module to manage various aspects of services for System V Init beyond what just the generic `service` module provides.

A couple of updates to module_utils in support of this.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

sysvinit

##### ANSIBLE VERSION

```
ansible 2.5.0 (sysvinit 4dbf2551cc) last updated 2018/01/16 15:44:01 (GMT -500)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/admiller/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/admiller/src/dev/ansible/lib/ansible
  executable location = /home/admiller/src/dev/ansible/bin/ansible
  python version = 2.7.14 (default, Dec 11 2017, 14:52:53) [GCC 7.2.1 20170915 (Red Hat 7.2.1-2)]

```

### CODE REVIEW

File: lib/ansible/modules/system/sysvinit.py [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+    # figure out started status, everyone does it different!
+    is_started = False
+
+    # user knows other methods fail and supplied pattern
+    if pattern:
+        is_started = get_ps(module, pattern)
+    else:
+        worked = False
+        if location.get('service'):
+            # standard tool that has been 'destandarized' by reimplementation in other OS/distros
```
Comment (bcoca) [inline lib/ansible/modules/system/sysvinit.py:185]: as standard as the 'ini' format ... it looks the same but its not!

Comment (maxamillion) [reply lib/ansible/modules/system/sysvinit.py:185]: @bcoca fortunately, through my testing on devuan and CentOS 6 it appears for the limited use cases covered by the module they do mirror each other's functionality

Comment (bcoca) [reply lib/ansible/modules/system/sysvinit.py:185]: its more 'esoteric' distros or 'non linuces' that worry me, but ... we can wait for feedback as i hope those targets have 'fixed' their divergences

Comment (maxamillion) [reply lib/ansible/modules/system/sysvinit.py:185]: @bcoca +1 - I honestly was under the impression that non linux systems completely lacked the service util, but that's probably an erroneous assumption to make

Comment (bcoca) [reply lib/ansible/modules/system/sysvinit.py:185]: i.e freebsd has had 'service' (iirc since 8.0) but it actually uses a rc init system, not sysV.

Comment (maxamillion) [reply lib/ansible/modules/system/sysvinit.py:185]: ah +1

**Comment by bcoca** [comment]
rebuild_merge

**Comment by maxamillion** [comment]
rebuild_merge

**Comment by maxamillion** [comment]
shippable failure due to dns resolution issue, not related to PR. merging.

**Pull request merged**